schemas/+lab/ManipulationType.m

```diff
@@ -0,0 +1,11 @@
+%{
+# Types of manipulation that can be performed in a experiment
+manipulation_type              : varchar(64)
+-----
+manipulation_description       : varchar(2555)
+%}
+
+classdef ManipulationType < dj.Lookup
```

Comment:
ManipulationType might not be necessary. To query a Session with optogenetics. We could simply do `Session & OptogeneticSession`

Reply:
I think the idea for the future is that this table could serve for other manipulations as well: 
e.g:
Muscimol Inactivation experiments
Electric stimulation, etc.

Reply:
session_manipulation table

Reply:
I added a **table_population**  (is a list of "main" tables to be populated with this manipulation)
Maybe not needed entirely but I added the same field for Task and I think is somewhat analog.